apps/hip/templates/includes/sidebar.html

```diff
@@ -45,8 +45,10 @@ <h5 class="is-size-7 desktop-hidden-hip">Response Partners</h5>
   </div>
   <ul class="mb-4 desktop-hidden-hip {% if hidden_desktop %}is-hidden-desktop{% endif %}">
     <li class="pl-4 pb-2 sidenav-link-hip" data-href="#closed-pod-login">
```

Comment:
I think we just need to toggle this section with `Partner Log out` if the user is authenticated and this will be good to go.

Reply:
Good 👁️ . I've updated it based on the mobile wireframe.